test/NuGet.Clients.Tests/NuGet.SolutionRestoreManager.Test/VsSolutionRestoreServiceTests.cs

```diff
@@ -1849,6 +1851,90 @@ public async Task NominateProjectAsync_CancelledToken_ThrowsOperationCanceledExc
             await Assert.ThrowsAsync<OperationCanceledException>(async () => await service.NominateProjectAsync(@"f:\project\project.csproj", projectRestoreInfo, CancellationToken.None));
         }
 
+        [Fact]
+        public void ToPackageSpec_CentralVersions_AreAddedToThePackageSpecIfCPVMIsEnabled()
+        {
+            // Arrange
+            ProjectNames projectName = new ProjectNames(@"f:\project\project.csproj", "project", "project.csproj", "prjectC");
+            var emptyReferenceItems = Array.Empty<VsReferenceItem>();
+
+            var targetFrameworks = new VsTargetFrameworkInfo3[] { new VsTargetFrameworkInfo3(
+                targetFrameworkMoniker: CommonFrameworks.NetStandard20.ToString(),
+                packageReferences: new[] { new VsReferenceItem("foo", new VsReferenceProperties()) },
+                projectReferences: emptyReferenceItems,
+                packageDownloads: emptyReferenceItems,
+                frameworkReferences: emptyReferenceItems,
+                projectProperties: new VsProjectProperty[] { new VsProjectProperty("ManagePackageVersionsCentrally", "true") },
+                centralPackageVersions: new[]
+                        {
+                            new VsReferenceItem("foo", new VsReferenceProperties(new []
+                            {
+                                new VsReferenceProperty("Version", "2.0.0")
+                            }))
+                        })
+            };
+
+            // Act
+            var result = VsSolutionRestoreService.ToPackageSpec(projectName, targetFrameworks, CommonFrameworks.NetStandard20.ToString(), string.Empty);
+
+            // Assert
+            var tfm = result.TargetFrameworks.First();
+
+            Assert.Equal(1, tfm.CentralPackageVersions.Count);
+            Assert.True(tfm.CentralPackageVersions.ContainsKey("foo"));
+            Assert.Equal("[2.0.0, )", tfm.CentralPackageVersions["foo"].VersionRange.ToNormalizedString());
+            Assert.Equal(1, tfm.Dependencies.Count);
+            Assert.Null(tfm.Dependencies.First().LibraryRange.VersionRange);
+            Assert.True(result.RestoreMetadata.CentralPackageVersionsEnabled);
```

Comment:
It's subjective, but maybe something like this is easier to read?
```suggestion
            var packageVersion = Assert.Single(tfm.CentralPackageVersions);
            Assert.Equal("foo", packageVersion.Id);
            Assert.Equal("[2.0.0, )", packageVersion.VersionRange.ToNormalizedString());
            
            var packageReference = Assert.Single(tfm.Dependencies);
            Assert.Null(packageReference.LibraryRange.VersionRange);
            
            Assert.True(result.RestoreMetadata.CentralPackageVersionsEnabled);
```

Reply:
Updated with the proposed Asserts.